### PR TITLE
Issue #173: Migration of HouseMemberDocumentController to OpenApi yaml schema

### PR DESCRIPTION
**Pull request by huyle101**
## 🚀 Description
Added OpenAPI spec for HouseMemberDocumentController

## 📄 Motivation and Context
Fixes Issue #173 

## 🧪 How Has This Been Tested?

- Test cases are passing

- Verified functionalities via running locally Home-Web UI

## 📦 Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [x] My code follows the code style of this project(Do your best to follow code styles. If none apply just skip this).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: api/src/main/resources/api.yaml [hunk 1]
```diff
@@ -82,6 +82,102 @@ paths:
                 $ref: '#/components/schemas/GetUserDetailsResponseUser'
         '404':
           description: "If userId is invalid"
+  /members/{memberId}/documents:
+    get:
+      tags:
+        - HouseMemberDocuments
```
Comment (mslowiak) [inline api/src/main/resources/api.yaml]: Let's change the tag name to `Documents` and list it in tags names in line 9

File: api/src/main/resources/api.yaml [hunk 2]
```diff
@@ -82,6 +82,102 @@ paths:
...
+      operationId: deleteHouseMemberDocument
+      parameters:
+        - in: path
+          name: memberId
+          schema:
+            type: string
+          required: true
+      responses:
+        '204':
+          description: id document deleted
```
Comment (mslowiak) [inline api/src/main/resources/api.yaml]: `id` -> `if`

File: api/src/main/resources/api.yaml [hunk 3]
```diff
@@ -82,6 +82,102 @@ paths:
...
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                memberDocument:
+                  type: string
+                  format: binary
+      responses:
+        '204':
+          description: If document updated
```
Comment (mslowiak) [inline api/src/main/resources/api.yaml]: `If`  -> `if` to be consistent

File: api/src/main/resources/api.yaml [hunk 4]
```diff
@@ -82,6 +82,102 @@ paths:
...
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                memberDocument:
+                  type: string
+                  format: binary
+      responses:
+        '204':
+          description: If document saved
```
Comment (mslowiak) [inline api/src/main/resources/api.yaml]: `If`  -> `if` to be consistent

**Review by mslowiak — CHANGES_REQUESTED** [review]
There are small changes so it should not take long to fix that.
However please use rebase rather than merge, thanks :)

**Comment by huyle1097** [comment]
> However please use rebase rather than merge, thanks :)

Hey @mslowiak 
Do I need to revert the merge commit then do the rebase?
After I commit the new changes, I try to rebase with _upstream/master_ but it looks so worst :disappointed: 
image
Sorry, I never use rebase before :crying_cat_face:

**Review by mslowiak — APPROVED** [review]

**Comment by mslowiak** [comment]
Thank you for your contribution @huyle1097 :)

**Comment by mslowiak** [comment]
@huyle1097 
For the future issues please include 'Closes: #173' instead of 'Fixes Issue #173'. That will automatically close issue when PR is merged :)

**Comment by huyle101** [comment]
Thanks @mslowiak

**Pull request merged**